ert3/workspace/_workspace.py

```diff
@@ -145,13 +145,15 @@ def _validate_ensemble_size(
         )
 
 
-def _validate_inputs(
+def _validate_stage(
     stage_config: ert3.config.StagesConfig, ensemble_config: ert3.config.EnsembleConfig
 ) -> None:
     stage_name = ensemble_config.forward_model.stage
     stage = next((stage for stage in stage_config if stage.name == stage_name), None)
     if stage is None:
-        raise ert.exceptions.ConfigValidationError(f"Invalid stage: '{stage_name}''.")
+        raise ert.exceptions.ConfigValidationError(
+            f"Invalid stage in forward model: '{stage_name}''."
+        )
```

Comment:
Could add the possible options here as well maybe? Something like, `f"Invalid stage in forward model: '{stage_name}''. Must be one of: " + ", ".join(stage.name for stage in stage_config)` ?

Reply:
Done.